apps/portal/templates/gadgets/igviz/js/main.js

```diff
@@ -14,7 +14,8 @@ if(type === "realtime") {
     columns = gadgetConfig.columns;
     //subscribe to websocket
     subscribe(datasource.split(":")[0], datasource.split(":")[1], '10', 'carbon.super',
```

Comment:
@suhothayan 
I checked the code and dashboard, this value is used to created the directory for domain where gadgets are stored. This does not work in tenant mode. I created a JIRA(CEP-1291) for above bug.